index_V1

```diff
@@ -11,6 +10,19 @@
 		
 		<link rel="stylesheet" href="css/style_index.css">
 		
+		<style>
+			
+			.circle_1:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_1:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
+			.circle_2:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_2:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
+			.circle_3:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_3:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
+			.circle_4:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
+			.circle_4:hover p {transform: scale(0.909, 0.909); transition: 0.6s;}
+			
+		</style>
+		
```

Comment:
Here I added the css styling code to make circle divs change size when user hovers.